README.md

```diff
@@ -2,65 +2,217 @@
 
 ## Overview
 
-You can use this document to add a SignalFx wrapper to your AWS Lambda for C#.
+You can use this document to add a SignalFx wrapper to your AWS Lambda for .NET.
 
-The SignalFx C# Lambda Wrapper wraps around an AWS Lambda C# function handler, which allows metrics to be sent to SignalFx.
+The SignalFx C# Lambda Wrapper wraps around an AWS Lambda .NET function handler, which allows metrics and traces to be sent to SignalFx.
 
-## Step 1: Install via NuGet
+At a high-level, to add a SignalFx .NET Lambda wrapper, you can:
+   * Package the code yourself; or
+   * Use a Lambda layer containing the wrapper, and then attach the layer to a Lambda function.
 
-1. Add the following package reference to `.csproj` or `function.proj`:
-```xml
-  <PackageReference Include="signalfx-lambda-functions" Version="2.0.1"/>
+To learn more about Lambda Layers, please visit the [AWS documentation site](https://docs.aws.amazon.com/lambda/latest/dg/configuration-layers.html).
+
+## Step 1: Add the wrapper to the project
+
+Add the [`signalfx-lambda-functions` NuGet package](https://www.nuget.org/packages/signalfx-lambda-functions/) to your project.
+
+For advanced users who want to reduce the size of deployment packages, you can use the package as a developer dependency, but in production, you would add the wrapper to a layer in the Lambda environment. This option allows you to work with the wrapper in a local setting and reduce the size of deployment packages at the same time.
```

Comment:
Does the process have instructions detailed anywhere that can be linked to?

Reply:
I'm lifting the legacy instructions and it is pretty similar for all wrappers, that said this is not ideal: it should be directly the CLI commands instead. Punting for now.